### ISSUE TITLE
Image Similarity Documentation is still unclear

### ISSUE THREAD

**Issue by lunardog** [apple/turicreate#2509]
> https://github.com/apple//turicreate/blob/master/src/python/turicreate/toolkits/image_similarity/image_similarity.py#L44

This sentence sounds like it's a patchwork of partial sentences:

```
   feature : string
        indicates that the SFrame has only column of Image type and that will
        Name of the column containing the input images. 'None' (the default)
        be used for similarity.
```

How should it read?

**Comment by lunardog** [comment]
My attempt to fix in PR #2510

**Comment by TobyRoseman** [comment]
Fixed by #2510
